blivet/devices/storage.py

```diff
@@ -809,6 +809,13 @@ def direct(self):
     def is_disk(self):
         return self._is_disk
 
+    @property
+    def is_empty(self):
+        if not self.partitioned:
+            return self.format.type is None
```

Comment:
I think you may need an exception for lvm vgs here, just in case someone accesses this property for one.

Reply:
Good point, I didn't realize there are devices without format that can be "not empty". Same applies to thinpools. I've added an extra check for number of children.